jumpscale/sals/reservation_chatflow/__init__.py

```diff
@@ -11,7 +11,7 @@
 import gevent
 
 
-def wait_http_test(url: str, timeout: int = 60, verify: bool = True) -> bool:
+def wait_http_test(url: str, timeout: int = 60, verify: bool = True, status_code=200) -> bool:
```

Comment:
can we use wait_http_test from nettools sal here instead of implementing the function?

Reply:
same as above, but here i check for status code 403 which the minio return code as it requires authentication